docs/excel2xml-module.md

```diff
@@ -258,6 +258,36 @@ make_text_prop(
 ```
 
 
+#### The Special Properties of `<video-segment>` and `<audio-segment>`
+
+`<video-segment>` and `<audio-segment>` work differently than normal resources.
+Not only do they have a dedicated tag (i.e., different from `<resource>`), 
+but their properties also have dedicated tags.
+
+Here's how you can create them in your Python script:
+
+```python
+from dsp_tools import excel2xml
+
+segment = excel2xml.make_video_segment("label of my video segment", "segment_1")
+segment.append(excel2xml.make_isSegmentOf_prop("video_resource_id"))
+segment.append(excel2xml.make_hasSegmentBounds_prop(start=0, end=10))
+segment.append(excel2xml.make_hasTitle_prop("title of the segment"))
+segment.append(excel2xml.make_hasDescription_prop("description of the segment"))
+segment.append(excel2xml.make_hasComment_prop("comment to the segment"))
+segment.append(excel2xml.make_hasKeyword_prop("keyword"))
```

Comment:
Is it also possible to enter a list? Or more than one element?

Reply:
Changed the example, so that it becomes clearer: https://github.com/dasch-swiss/dsp-tools/commit/16aaba3cbcd1c898371a6760e8f87222f4dd41ab